cax-ap-cam/cax_ap_cam/const.py

```diff
@@ -0,0 +1,25 @@
+from siriuspy.envars import VACA_PREFIX as _vaca_prefix
```

Comment:
This module can be renamed to `csdev`, as we generally use in other IOCs

Reply:
agreed.